<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>networkcreateDialogBase</class>
 <widget class="QDialog" name="networkcreateDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Network_creator</string>
  </property>
  <widget class="QTabWidget" name="qtab">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>381</width>
     <height>301</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QPushButton" name="createnetwork">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>200</y>
       <width>161</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Create</string>
     </property>
    </widget>
    <widget class="QPushButton" name="runButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>200</y>
       <width>161</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Run</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>170</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Create network:</string>
     </property>
    </widget>
    <widget class="QLabel" name="Buildingconnection">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Building connection:</string>
     </property>
    </widget>
    <widget class="QLabel" name="Selectr">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Select road layer:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="buildingsComboBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>361</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="heatingComboBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>361</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="roadsComboBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>361</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>100</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Select heating layer:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Select buldings layer:</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Add number of house:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Simultaneity factor:</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Add cumulative peak load:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>130</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Load at node:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="cumulativePeakLoadLineEdit">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>50</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="numHousesLineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="simultaneityFactorLineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="loadAtNodeLineEdit">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>150</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="simultaneityFactorButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Run</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushCumulativeButton">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>80</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Run</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Mass flow rate(kg/s):</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="massFlowLineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushMassflowButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Run</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
